### PR TITLE
EUTXO paper: back to 15 pages

### PR DESCRIPTION
**Pull request by omelkonian**
* Also add Makefile for explicit build instructions.

### CODE REVIEW

File: papers/eutxo/Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+MAIN=eutxo
+
+$(MAIN).pdf:
+	xelatex $(MAIN).tex
```
Comment (michaelpj) [inline papers/eutxo/Makefile]: I don't know how much this helps them, since most people don't use `xelatex` ;) But it gives the idea, I guess.

Comment (omelkonian) [reply papers/eutxo/Makefile]: I will replace it with `pdflatex`, but I do not like the font thickness that much (it's quite less boldy).

Comment (michaelpj) [reply papers/eutxo/Makefile]: I don't think it matters: they're going to recompile it, and the one that's hosted by us is built with `pdflatex` anyway.

Comment (omelkonian) [reply papers/eutxo/Makefile]: Actually, the Makefile is useless in this repo, I will just remove it.

**Pull request merged**